chrome/js/helpers.js

```diff
@@ -1,19 +1,28 @@
 /**
  * Returns the highlighted text from a webpage.
- * @argument {string, number, number} 
+ * @argument {string, number, number}
  * String of the highlighted text, 2nd and 3rd arguments are the coordinates for the element
  * @return {null} String of the highlighted text.
  */
 const insertDiv = (text, x, y) => {
     var div = document.createElement('div');
     div.innerHTML = text
     div.className = 'text'
+    div.style.height = '500'
```

Comment:
I think we shouldn't modify the styling directly, see on line 10 the className change? there's a css file loading with that style. Just edit that class instead